product-scenarios/1-manage-public-partner-private-apis/1.1-expose-service-as-rest-api-and-apply-qos/1.1.2-create-rest-api-by-import-oas-document/src/test/java/org/wso2/am/scenario/tests/rest/api/creation/RESTApiCreationUsingOASDocTestCase.java

```diff
@@ -15,15 +15,22 @@
 */
 package org.wso2.am.scenario.tests.rest.api.creation;
 
-import org.testng.annotations.AfterTest;
+import com.sun.xml.bind.v2.TODO;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 import org.wso2.am.integration.test.utils.APIManagerIntegrationTestException;
-import org.wso2.am.scenario.test.common.APIPublisherRestClient;
-import org.wso2.am.scenario.test.common.APIRequest;
-import org.wso2.am.scenario.test.common.ScenarioTestBase;
+import org.wso2.am.scenario.test.common.*;
```

Comment:
Shall we keep specific imports only without *

Reply:
Sure. Fixed with cd8f6d11c67af9a507d775a221e0c9a611bac79f